pcapgo/capture_test.go

```diff
@@ -0,0 +1,34 @@
+package pcapgo_test
+
+import (
+	"log"
+	"os"
+
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
+	"github.com/google/gopacket/pcapgo"
+)
+
+func Example_captureEthernet() {
+	f, err := os.Create("/tmp/lo.pcap")
```

Comment:
I've never seen this before... why does it work?  Why does this file translate to net.InterfaceByName working?  Inquiring minds like mine want to know :)

Reply:
Not sure I follow. The filename has nothing to do with the rest of the code? NewEthernetHandle takes an ifname, which is passed to net.InterfaceByName, and we specify ifname="lo".